etc/profile-a-l/keepassxc.profile

```diff
@@ -93,6 +93,7 @@ private-etc
 private-tmp
 
 dbus-user filter
+dbus-user.own org.freedesktop.secrets
```

Comment:
One final remark. As hinted at in my previous comment there's a difference between dbus-user.`own` foo and dbus-user.`talk` foo. KeePassXC only needs to _talk_ to the Secret Service, so drop this line and add `dbus-user.talk org.freedesktop.secrets` at the bottom of this D-Bus options block (to keep options sorted alphabetically).

Reply:
The Secret Service functionality of KeePassXC doesn't work with `dbus-user.talk`. [KeePassXC offers the Secret Service](https://github.com/keepassxreboot/keepassxc/blob/a31c5ba006198f2fdadfd6c4f587cfed602c81c6/src/fdosecrets/dbus/DBusMgr.cpp#L406-L428). It therefore needs `dbus-user.own`. When other programs talk to `org.freedesktop.secrets`, then end up at KeePassXC, which manages the secrets for them.

Reply:
Thanks for the detailed info. All good then as is!